### PR TITLE
feature/i148 stateful buttons

### PR DESCRIPTION
**Pull request by carlo**
See #148.

### CODE REVIEW

**Comment by carlo** [comment]
@fspoettel: Before you ask: I went with `<button/>` over the `<Button/>` component because the I didn't see the point of invoking another component for what is basically just a dumb container tag.

**Comment by fspoettel** [comment]
@carlo basebranch is `feature/lds-2.3.1`. I changed it for this PR

**Comment by carlo** [comment]
@fspoettel: Hnnnnnngh, okay. Changed code accordingly.

File: docs/src/app/pages/StatefulButtons/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import DecoratorList from '../../components/DecoratorList';
+import HeaderIcon from '../../components/HeaderIcon';
+import Masthead from '../../components/MastHead';
+import PropTypeDescription from '../../components/PropTypeDescription';
+
+import ExampleSelected from './ExampleSelected';
+import ExampleUnselected from './ExampleUnselected';
+
+const StatefulButtons = () =>
+  <div>
```
Comment (fspoettel) [inline docs/src/app/pages/StatefulButtons/index.js]: Every time you don't wrap multiline JSX, a cute kitten drowns in its own tears.

File: src/components/Menu/DropdownMenu.js [hunk 2]
```diff
@@ -161,4 +161,74 @@ export class DropdownMenu extends Component {
   }
 }
 
+DropdownMenu.contextTypes = { cssPrefix: PropTypes.string };
```
Comment (fspoettel) [inline src/components/Menu/DropdownMenu.js]: This doesnt seem right

File: src/components/Menu/Picklist.js [hunk 3]
```diff
@@ -88,4 +88,35 @@ export class Picklist extends React.Component {
   }
 }
 
+Picklist.contextTypes = { cssPrefix: PropTypes.string };
```
Comment (fspoettel) [inline src/components/Menu/Picklist.js]: ☝

**Pull request closed without merging**